src/basic/cgroup-util.c

```diff
@@ -2376,7 +2376,7 @@ int cg_mask_supported(CGroupMask *ret) {
         /* BPF-based pseudo-controllers */
         r = cg_unified_controller(SYSTEMD_CGROUP_CONTROLLER);
         if (r > 0)
-                mask |= CGROUP_MASK_BPF_FIREWALL;
+                mask |= CGROUP_MASK_BPF_FIREWALL | CGROUP_MASK_BPF_DEVICES;
```

Comment:
this one should probably have bpf_devices_supported() or so, which does explicit changes to see if this functionality is available...